packages/connect/src/utils.ts

```diff
@@ -1,7 +1,3 @@
 export function getStacksProvider() {
   return window.StacksProvider || window.BlockstackProvider;
```

Comment:
We are no longer injecting `BlockstackProvider`
```suggestion
  return window.StacksProvider;
```